ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/QueryStreamHandler.java

```diff
@@ -164,7 +165,9 @@ private CommonRequest getRequest(final RoutingContext routingContext) {
       if (!ksqlRequest.isPresent()) {
         return null;
       }
-      sql = ksqlRequest.get().getKsql();
+      // Set masked sql statement if request is not from OldApiUtils.handleOldApiRequest
+      ApiServerUtils.setMaskedSql(ksqlRequest.get());
```

Comment:
Should we use `setMaskedSqlIfNeeded()` here instead, since it may already be set?